design/use_cases/_template_use_case.md

```diff
@@ -8,9 +8,14 @@ As a \<Developer, Researcher, Technician, etc.> I want the robot to \<action> so
 
 # Example:
 
-# Warehouse Navigation
-As a warehouse robot operator, I want the robot to navigate without colliding into people or objects so that it doesn't hurt anyone or damage anything
+# Collision Avoidance
+As a robot user, I want the robot to navigate without colliding into people or objects so that it doesn't hurt anyone or damage anything
 
 ## More details
- - I want this so that I know the robot won't damage itself, damage property or hurt anyone
- - I shouldn't have to interact with the robot to prevent it from crashing into people or things
+ - Why is this needed?
+   - I want this so that I know the robot won't damage itself, damage property or hurt anyone
+   - Example: a logistics robot in a warehouse must avoid shelves, people, forklifts, and other robots
+ - What is the expected user interaction? 
+   - I shouldn't have to interact with the robot to prevent it from crashing into people or things
+- Are there any non-functional requirements? (build system, tools, performance, etc)
+   - The performance needs to be fast enough to avoid moving objects such as people walking or other moving robots
```

Comment:
Additional optional sections to consider adding to the uses cases:
- Technical Variations
- Exception conditions.

For technical variations, I can see variability based on sensor suite and motion platform in some use cases.

Reply:
@crdelsey - do you mean that if a particular sensor or motion platform is used, the use case changes? I think we want to keep it abstract enough to work with more than one set of sensors or motion platforms. Maybe I don't understand what you're suggesting here.

Reply:
@mkhansen-intel At the level of detail these use cases are at, the platform and sensors probably don't come into play to much. However, when we try to go to the next level ...  for instance, defining what exactly we mean by avoiding an obstacle, well, then it probably matters that our sensors can only see in a narrow field of view, or we have no ability to look up or down.

Even at our current level of detail, the technical variations section may be applicable to the "keep out zones" use case. The main use case could specify that the user defines the keep out zones in a gui. A technical variation could be that the user specifies the keep out zone using an API or through markers on a map or markers in the real world (eg tape on the ground, etc)